crates/rspack_plugin_html/src/config.rs

```diff
@@ -131,4 +132,14 @@ impl HtmlPluginConfig {
         .render(compilation, filename),
     }
   }
+  pub fn get_relative_path(&self, compilation: &Compilation, filename: &str) -> String {
```

Comment:
sugar_path already support this

Reply:
It is not just to get the relative path between two PathBufs. This function is used to get the relative path to the project root directory, so exists favicon logic can be reused.